testing/src/main/java/org/oppia/android/testing/profile/ProfileTestHelper.kt

```diff
@@ -56,6 +56,23 @@ class ProfileTestHelper @Inject constructor(
     return result
   }
 
+  fun addLongNameProfile(): LiveData<AsyncResult<Any?>> {
+    profileManagementController.addProfile(
+      name = "Natrajan Subramanniyam Balaguruswamy",
+      pin = "123",
+      avatarImagePath = null,
+      allowDownloadAccess = false,
+      colorRgb = -10710042,
+      isAdmin = false
+    )
+    val result = profileManagementController.loginToProfile(
+      ProfileId.newBuilder().setInternalId(0)
```

Comment:
Will this be the correct ID if the profile above isn't the first one added?

Reply:
Actually, this isn't the first ID that is being added, I am calling this method after `initializeProfiles()`, so this is the order of the IDs getting added - 

1. Admin
2. Ben
3. Nikita
4. Natrajan Subramanniyam Balaguruswamy

Reply:
It was wrong actually, I was logging in with `Admin` profile but it didn't affect the test cases because I used the correct profiled to launch the `HomeActivity`

Reply:
I think logging into admin by default is probably correct for this method, then.